_includes/search.html

```diff
@@ -23,7 +23,21 @@
                 <div class="collapse" id="filter-options" aria-expanded="true">
                     <div class="card card-body">
                         <p>Tags</p>
-                        <div class="tags"></div>
+                        <div class="tags">
+                          {% assign tags_to_capitalize = "aws,gcp,azure,gke,eks" | split: "," %}
```

Comment:
to capitalize as in `AZURE` or as in `Azure`?

Reply:
Oh, whoops! Good catch. Will fix in another PR.